src/metabase/util/date_2.clj

```diff
@@ -106,13 +106,22 @@
                     (.toFormatter builder))
    OffsetTime     (let [builder (doto (DateTimeFormatterBuilder.)
                                   (.append (DateTimeFormatter/ofLocalizedTime FormatStyle/MEDIUM))
-                                  (.appendPattern " (OOOO)"))]
+                                  (.appendLiteral " (")
+                                  (.appendLocalizedOffset TextStyle/FULL)
+                                  (.appendLiteral ")"))]
                     (.toFormatter builder))
    OffsetDateTime (let [builder (doto (DateTimeFormatterBuilder.)
                                   (.appendLocalized FormatStyle/LONG FormatStyle/MEDIUM)
-                                  (.appendPattern " (OOOO)"))]
+                                  (.appendLiteral " (")
+                                  (.appendLocalizedOffset TextStyle/FULL)
+                                  (.appendLiteral ")"))]
```

Comment:
this code does the same thing but I think `.appendLocalizedOffset` is a little clearer than `(.appendPattern "OOOO")` so I made this unrelated change here